index.js

```diff
@@ -54,6 +54,12 @@ class ScmRouter extends Scm {
      * @param  {Object}         options      settings for scm module
      */
     loadPlugin(plugin, options) {
+        if (plugin === 'router') {
```

Comment:
Because it would be terrible if specified the `router` to the `plugin` as follows:
```
scms:
    plugin: router
```